frontend/public/components/utils/alerts.tsx

```diff
@@ -23,9 +25,9 @@ export const ExpandableAlert: React.FC<CustomAlertProps> = ({ alerts, variant })
       className="co-alert"
       title={
         <>
-          {`There are ${alertCount} ${variant} alerts.`}
+          {t('utils~There is {{count}} {{variant}} alert.', { count: alertCount, variant })}
```

Comment:
I would use the `public` namespace throughout

```suggestion
          {t('public~There is {{count}} {{variant}} alert.', { count: alertCount, variant })}
```